packages/block-editor/src/components/block-list/block-wrapper.js

```diff
@@ -199,7 +202,9 @@ const BlockComponent = forwardRef(
 				{ ...props }
 				id={ blockElementId }
 				ref={ wrapper }
-				className={ classnames( className, props.className ) }
+				className={ classnames( className, props.className, {
+					'wp-block': ! supportAlignments,
```

Comment:
Why does this class have to be removed when there's alignment? Does it hurt to have .wp-block > .wp-block?